tests/storage/cdi_import/test_import_http.py

```diff
@@ -542,6 +542,7 @@ def test_blank_disk_import_validate_status(data_volume_multi_storage_scope_funct
     ],
 )
 def test_vmi_image_size(
+    xfail_test_if_gcp_sc,
```

Comment:
For the 13Gi param - this test can still run
What should happen if we requested 64M, but got 4Gi PVC? Should VMI see the 4Gi?